App.js

```diff
@@ -61,9 +60,58 @@ const AppContainer = createAppContainer(TabNavigator);
 
 export default class App extends Component<Props> {
 
+  events = ["event 2", "event 4", "event 5"];
+
+  async checkPermissions() {
+    const enabled = await firebase.messaging().hasPermission();
+    if (enabled) {
+      console.log("Permissions Enabled");
+    } else {
+      console.log("Asking for permissions");
+      this.askPermissions();
+    }
+  }
+
+  async askPermissions() {
+    try {
+      await firebase.messaging().requestPermission();
+      console.log("Permissions enabled");
+    } catch (error) {
+      console.log("User rejected permissions");
+    }
+  }
+
+  componentDidMount() {
+    this.removeNotificationListener = firebase.notifications().onNotification((notification: Notification) => {
+        // Process your notification as required
+        const { title, body } = notification
+        console.log(this.events.some((event) => {
```

Comment:
one of the reasons schedule should be locally triggered - `this.events` is not clearly defined and a little unwieldy to hold in memory. If buzzer wanted to reference these events, it should probably specify event id. 

That aside though, you're also looking to reference the user's tags, not the event. 
the check should be `this.followedTags.some(tag => notif.data.tags.includes(tag));` you forgot to return here btw
You can hardcode the list of tags for this PR.

Reply:
So I was a bit confused with this. Would people be subscribing to certain tags or events or both? And then in buzzer I'm currently working it so that you specify the tag/tags associated with it.